packages/components/src/components/date-picker/date-picker.css

```diff
@@ -49,7 +49,7 @@ duet-date-picker .duet-date__input {
   background-color: var(--telekom-color-ui-state-fill-standard);
   padding: 1.125rem calc(2rem - var(--telekom-spacing-composition-space-01))
     0.25rem calc(var(--spacing-x) - var(--telekom-spacing-composition-space-01));
-  height: var(--telekom-spacing-composition-space-13);
+  /* height: var(--telekom-spacing-composition-space-13); */
```

Comment:
is this intended? Visual tests failed because of this…

Reply:
oh no i was testing something, thanks